### ISSUE TITLE
Does this work with a ARM Mac?

### ISSUE THREAD

**Issue by JonEHolland** [Alban098/NEmuS#1]
When I build it using JDK 20 (and adding necessary module nonsense to make JavaFX happy), I just a zombie process with no JavaFX window.

I'll submit a PR if I figure out why this is happening, but if you have any ideas let me know.

**Comment by Alban098** [comment]
At the moment I can't pinpoint a specific culprit.
I would look at OpenGL, I remember having trouble running it on my Mac at the time.
Another suspect could be the Java version, I did it using JDK 11, and only brefely tested it with 16, 2 years later when introducing maven.

Honestly I started this as a small experiment out of boredom during my studies, with little to no experiences on how to make a well structured and maintainable project.
Therefore I did not plan for it to support multiple platforms, especially not ARM based chips as they wheren't even a thing yet.
